packages/react/tsconfig.build.json

```diff
@@ -15,6 +15,7 @@
     "src/utils/story-helpers.tsx",
     "**/*.stories.tsx",
     "**/*.test.tsx",
+    "src/**/*.figma.tsx",
```

Comment:
Quick question, do the files for figma need to be included in the package or can they be used on push in the actions workflow and then discarded for the package build? 👀

Reply:
They do NOT need to be included in the package.

They are also not really interpreted but only "read" by the figma tool, which is why it may be hard to get them 100% TS compatible.